db/migrate/20150818181426_add_divisible_to_tenants.rb

```diff
@@ -0,0 +1,5 @@
+class AddDivisibleToTenants < ActiveRecord::Migration
+  def change
+    add_column  :tenants, :divisible, :boolean, :default => true
```

Comment:
@Fryguy Can we do both? I'm only asking because with :default => true in the migration, we don't have to worry about adding additional code in the migration to set divisible to true on existing tenants. The migration will do that automatically.